kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java

```diff
@@ -1223,20 +1225,87 @@ public void testFindTaskEvents() throws Exception {
 
             TaskSummary taskInstance = tasks.get(0);
 
+            TaskEventInstance expectedTaskEventInstance = TaskEventInstance
+                    .builder()
+                    .type(TaskEvent.TaskEventType.ADDED.toString())
+                    .processInstanceId(processInstanceId)
+                    .taskId(taskInstance.getId())
+                    .user(PROCESS_ID_USERTASK)
```

Comment:
it is because added task is a system operation, logged system user who performs the operation